hs_docker_base

```diff
@@ -1 +1 @@
-Subproject commit 0b178b09012f38773a300bc290d4894074d4b3bc
+Subproject commit 39bfe8c29860e008fb8e975eb5d2b1b4ee88ff57
```

Comment:
@ironfroggy I thought if you pull the latest changes, and commit hs_docker_base, it should have the same SHA as the latest develop, not sure what happened then. @mjstealey probably has better ideas on this.